server/fleet/software.go

```diff
@@ -136,6 +136,8 @@ type SoftwareTitle struct {
 	Name string `json:"name" db:"name"`
 	// Source is the source reported by osquery.
 	Source string `json:"source" db:"source"`
+	// Browser is the browser type (e.g., "chrome", "firefox", "safari")
+	Browser string `json:"browser" db:"browser"`
```

Comment:
sanity checking: should we `omitempty`?

Reply:
gotcha, sanity checking that this also means we'll include a `browser` key for all kinds of software, is that right?

Reply:
thank you!